src/main/resources/application.yml

```
@@ -12,7 +12,21 @@ spring:
 auth:
   login_success:
-    base_uri: ${LOGIN_SUCCESS_BASE_URI}
\ No newline at end of file
+    base_uri: ${LOGIN_SUCCESS_BASE_URI}
+
+weather:
+  api:
+    key: ${OPENWEATHER_API_KEY}
+  scheduler:
+    # 초 분 시 일 월 요일
+    cron: 0 0 * * * *
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)